corva/types.py

```diff
@@ -1,5 +1,6 @@
-from typing import List, Union
+from typing import Any, Callable, List, TypeVar, Union
 
+_T = TypeVar('_T')
```

Comment:
what is this? why do we need it?

Reply:
i've deleted this, and also replaced type annotations with simple Callable